test/EndToEndTests/Services/ServiceWrapperApp/ServiceWrapperApp.cs

```diff
@@ -67,6 +67,8 @@ static int Main(string[] args)
             {
                 Console.WriteLine(e.Message);
                 PrintHelp();
+
+                return 1;

```

Comment:
How about make this 2 so you have a unique return code here, might be useful later on the calling side?

Reply:
Your change certainly covers the case I asked for :)

Reply:
Wanted to polish the code a little bit more while I was in there. :)